src/lib/uniforms-surfnet/src/OptGroupField.tsx

```diff
@@ -45,23 +45,25 @@ function OptGroup({
             description={intl.formatMessage({ id: `forms.fields.${name}.info` })}
             className={`${className} optgroup-field`}
         >
-            <EuiFlexItem>
-                <EuiFormRow
-                    error={false}
-                    isInvalid={false}
-                    id={name} // Not sure if this is always unique...
-                >
-                    <BoolField name="enabled" readOnly={readOnly} />
-                </EuiFormRow>
-            </EuiFlexItem>
-            {enabled &&
-                fields
-                    ?.filter((field) => field !== "enabled")
-                    .map((field) => (
-                        <EuiFlexItem key={field}>
-                            <AutoField name={field} {...itemProps} />
-                        </EuiFlexItem>
-                    ))}
+            <>
```

Comment:
Just curious - what does this do/solve?

Reply:
That's a React Fragment, when only one element is allowed, you can wrap multiple elements in a Fragment. You don't get an extra wrapped dom node.

https://reactjs.org/docs/fragments.html